woocommerce-services.php

```diff
@@ -1369,6 +1370,25 @@ function enqueue_wc_connect_script( $root_view, $extra_args = array() ) {
 				</div>
 			<?php
 		}
+
+		function render_schema_notices() {
+			$schemas = $this->get_service_schemas_store()->get_service_schemas();
+			if ( empty( $schemas ) || ! property_exists( $schemas, 'notices' ) || empty( $schemas->notices ) ) {
```

Comment:
A very small nitpick, but do we need `property_exists( $schemas, 'notices' )` considering that we will have to deploy the server first anyway?